app/steps/medicaid/intro_citizen.rb

```diff
@@ -2,5 +2,6 @@
 
 module Medicaid
   class IntroCitizen < Step
+    step_attributes(:us_citizen)
```

Comment:
this may be looking too far ahead, but later this will be about whether everyone in the household is a citizen. in snap apps we call this `everyone_a_citizen` -- should we use that here, too?